data/oot/world/overworld.yml

```diff
@@ -469,7 +469,7 @@
   exits:
     "Goron City": "true"
     "Dodongo Cavern": "has_bombflowers || is_adult"
-    "Kakariko": "setting(kakarikoGate, open) || event(KAKARIKO_GATE_OPEN)"
+    "Kakariko": "event(KAKARIKO_GATE_OPEN) || trick(OOT_PASS_COLLISION)"
```

Comment:
This is incorrect - if you have the gate setting set to open, you don't need the event (the event wouild require that you visited kak already)